### PR TITLE
Add ocr-numbers exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/ocr-numbers/Example.fs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+let matchCharacter input = 
+    match Map.tryFind input digitPatterns with
+    | Some x -> x
+    | None -> "?"
+
+let rows (input:string) = 
+    input.Split('\n')
+    |> Array.chunkBySize characterHeight
+
+let rowToCharacters (row: string[]) =
```
Comment (jwood803) [inline exercises/ocr-numbers/Example.fs]: I'm curious, will this work without having to specify what type it is and see if it can infer it?

Comment (ErikSchierboom) [inline exercises/ocr-numbers/Example.fs]: Yes this works :) The code works both locally in VS and remote on Travis.

Comment (jwood803) [inline exercises/ocr-numbers/Example.fs]: Sweet! Just something I noticed when messing with some of my stuff. The type inference is really good!

Comment (ErikSchierboom) [inline exercises/ocr-numbers/Example.fs]: Absolutely! Sometimes I find myself just trying things out, removing types, to see if it still compiles, and often it still does!

**Pull request merged**